libs/platform/src/lib/components/menu/menu-component/menu.component.scss

```diff
@@ -1,6 +1,16 @@
+@import '~fundamental-styles/dist/menu';
+
+$fd-menu-background-color: #ffffff;
```

Comment:
why do we have hardcoded color values?

Reply:
Hi Deno,
I followed the last committed changes by Kevin(https://github.com/SAP/fundamental-ngx/pull/1460#issuecomment-548873953) in PR https://github.com/SAP/fundamental-ngx/pull/1460 that addressed the SCSS changes as per your comments there. 
Have I followed it incorrectly?

Reply:
Would you rather I follow https://github.com/SAP/fundamental-styles/wiki/SASS-and-CSS-Custom-Properties#theme-parameters only for everything and only if there is a color not available in the SAP theming set, then use it in the way I have done it now?

Reply:
@droshev, I am using the theming parameters but to support the colors in IE, I am using it like this:
`$fd-menu-item-focus-color: #085caf;`
`outline: var(--sapButton_Emphasized_Hover_BorderColor, $fd-menu-item-focus-color) dotted 1px;`
Is this incorrect?